docs/history/changelog.md

```diff
@@ -8,40 +8,61 @@ Per the [normative and non-normative content and changes policy](https://docs.go
-  * [#980](https://github.com/open-contracting/standard/pull/980) Remove references to ☆ levels.
-  * [#1017](https://github.com/open-contracting/standard/pull/1017) Add link to bulk downloads in Guidance section.
+* Primer:
+  * Rewrite the Getting Started section as a Primer [#1298](https://github.com/open-contracting/standard/pull/1298) [#1302](https://github.com/open-contracting/standard/pull/1302) [#1365](https://github.com/open-contracting/standard/pull/1365)[#1387](https://github.com/open-contracting/standard/pull/1387).
 
-* Guidance section:
-  * [#1345](https://github.com/open-contracting/standard/pull/1345) Update legal references in pre-gualification guidance.
+* Guidance:
   * [#986](https://github.com/open-contracting/standard/pull/986) Add implementation guidance from OCP website.
-  * Add worked examples for the Map phase [#947](https://github.com/open-contracting/standard/pull/947) [#948](https://github.com/open-contracting/standard/pull/948) [#950](https://github.com/open-contracting/standard/pull/950) [#974](https://github.com/open-contracting/standard/pull/974) [#990](https://github.com/open-contracting/standard/pull/990) [#999](https://github.com/open-contracting/standard/pull/999) [#1007](https://github.com/open-contracting/standard/pull/1007) [#1123](https://github.com/open-contracting/standard/pull/1123).
-  * Add worked examples for the Build phase [#951](https://github.com/open-contracting/standard/pull/951) [#997](https://github.com/open-contracting/standard/pull/997).
-  * [#963](https://github.com/open-contracting/standard/pull/963) Remove guidance on web discovery.
-  * [#986](https://github.com/open-contracting/standard/pull/986) Merge Registration page into Build page.
-  * [#986](https://github.com/open-contracting/standard/pull/986) [#1012](https://github.com/open-contracting/standard/pull/1012) Merge Publication policy and Licensing pages into Publish page.
-  * [#1013](https://github.com/open-contracting/standard/pull/1013) Replace guidance on publication levels.
-  * [#1016](https://github.com/open-contracting/standard/pull/1016) Add links to OCDS profiles and linked standards.
-  * [#1150](https://github.com/open-contracting/standard/pull/1150) Align language in Build phase with language in Map phase.
-  * [#1193](https://github.com/open-contracting/standard/pull/1193) Removed language distinguishing "core", "community", and "local" extensions.
-  * [#1337](https://github.com/open-contracting/standard/pull/1337) Improve the structure of the Map page.
-  * [#1338](https://github.com/open-contracting/standard/pull/1338) Improve the structure of the worked examples on the Map page.
-  * [#1332](https://github.com/open-contracting/standard/pull/1332) Improve guidance on framework agreements.
   * [#1386](https://github.com/open-contracting/standard/pull/1386) Add link to OCDS Implementation Checklist.
+  * Map:
+    * Add examples [#1338](https://github.com/open-contracting/standard/pull/1338):
```

Comment:
@jpmckinney do we want to change the order now?

Reply:
What would we change it to? Right now it's just in PR order.

Reply:
Anyway, I'll merge as approved. We can change the log in another PR.